fastlane/lib/versions.rb

```diff
@@ -1,27 +1,3 @@
-# Gets the version, be it from Travis, Testflight, or Google Play
-def current_build_number(**args)
-	return build_number if build_number
-
-	begin
```

Comment:
For posterity's sake – we haven't used this chunk of code (the `begin`/`end` block) since … oh gosh, at least a while _before_ we left Travis.